### PR TITLE
podman 4.6.2

### PR DESCRIPTION
**Pull request by crunchtime-ali**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.

### CODE REVIEW

**Comment by chenrui333** [comment]
linux build failure

```
  ==> Checking out tag v4.6.2
  /usr/bin/env git checkout -f v4.6.2 --
  HEAD is now at 5db42e868 Bump to v4.6.2
  /usr/bin/env git reset --hard v4.6.2 --
  HEAD is now at 5db42e868 Bump to v4.6.2
  Error: An exception occurred within a child process:
    RuntimeError: /home/linuxbrew/.linuxbrew/opt/berkeley-db@5 not present or broken
```

**Comment by ZhongRuoyu** [comment]
Seems to be fixed by https://github.com/Homebrew/brew/pull/15920. (Same for #140766 and #140708.)

File: Formula/p/podman.rb [hunk 1]
```diff
@@ -37,8 +37,10 @@ class Podman < Formula
     depends_on "fuse-overlayfs"
     depends_on "gpgme"
     depends_on "libseccomp"
+    depends_on "mpdecimal"
     depends_on "slirp4netns"
     depends_on "systemd"
+    depends_on "unzip"
```
Comment (ZhongRuoyu) [inline Formula/p/podman.rb:43]: Were these deps added because of the previous job failures? They don't look related.

Comment (p-linnane) [reply Formula/p/podman.rb]: Yes, CI was throwing errors that they were missing. We should remove them and rerun now.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**